### ISSUE TITLE
RSS RFC-822 standard not valid for "fr" locale

### ISSUE THREAD

**Issue by frnode** [TryGhost/Ghost#13832]
### Issue Summary

Hello,

I use Ghost CMS with the "French" ("fr") language, I wanted to implement the RSS feed available here: https://ghost.org/integrations/custom-rss/

But in French language it does not respect the "[RFC-822](https://www.rssboard.org/rss-validator/docs/error/InvalidRFC2822Date.html)" standard, because it translates the date into the defined language and there is no way to force the display in "English" mode.

If anyone has any idea how we can get around this so that we have a valid feed!

Thank you,
Regards

### Steps to Reproduce

1. [Implement RSS](https://ghost.org/integrations/custom-rss/)
2. Change lang to "fr" in Admin
3. Go to the RSS page
4. Look <lastBuildDate> and <pubDate>

### Ghost Version

4.24

### Node.js Version

14.18.2-0

### How did you install Ghost?

Own server with https://hub.docker.com/r/bitnami/ghost/

### Database type

Other

### Browser & OS version

All

### Relevant log / error output

```shell
<lastBuildDate>jeu., 02 déc. 2021 19:17:18 +0100</lastBuildDate>
<pubDate>mar., 30 nov. 2021 08:30:00 +0100</pubDate>

Good format is:

<pubDate>Wed, 02 Oct 2002 15:00:00 +0200</pubDate>
<pubDate>Wed, 02 Oct 2002 15:00:00 +0200</pubDate>
```

[ Feed Validation Service ](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Finsys.fr%2Fblog%2Frss%2F)

### Code of Conduct

- [x] I agree to be friendly and polite to people in this repository

**Comment by aq1** [comment]
True, true. Hoped there is solution but I guess there isn't

**Comment by touzoku** [comment]
@ErisDS any chance to get #13892 merged? Our RSS is a mess 😿

**Comment by ErisDS** [comment]
This has been fixed by @touzoku in PR #13892. The fix will go out in the next release, which is currently our standard Friday afternoon automated release.
